Altis_Life.Altis/core/items/fn_boltcutter.sqf

```diff
@@ -6,8 +6,14 @@
     Description:
```

Comment:
This conflicts with the federal reserve. When trying to use boltcutters, nothing happens. I then replaced the file with the default fn_boltcutters file and it works.